reproject/mosaicking/coadd.py

```diff
@@ -82,6 +85,18 @@ def reproject_and_coadd(
         If `None`, the background matching will make it so that the average of
         the corrections for all images is zero. If an integer, this specifies
         the index of the image to use as a reference.
+    final_array : array or None
+        The final output array.  Specify this if you already have an
+        appropriately-shaped array to store the data in.  Must match shape
+        specified with `shape_out` or derived from the output
+        projection.
+    final_footprint : array or None
+        The final output footprint array.  Specify this if you already have an
+        appropriately-shaped array to store the data in.  Must match shape
+        specified with `shape_out` or derived from the output projection.
+    block_sizes : list of tuples or None
+        The block size to use for each cube (optional; meant for dask use)
```

Comment:
Don't mention 'cube' here, maybe just input dataset? And don't need to mention dask here, block sizes can be used even with numpy input. You can just say that these are passed to the ``block_size`` argument of the individual reproject functions.

Reply:
done